playgrounds/vite/src/Liquidity/index.tsx

```diff
@@ -79,12 +79,27 @@ export default function Liquidity({
   }, [nativeAssetValue, otherAssetValue, pluginMode]);
 
   const handleWithdraw = useCallback(async () => {
-    const ethAddress = await skClient?.thorchain.savings({
-      assetValue: nativeAsset!,
-      percent: withdrawPercent,
-      type: 'withdraw'
-    })
-    console.log("🚀 ~ handleWithdraw ~ ethAddress:", ethAddress)
+    let tx ;
```

Comment:
would be better to have const inside the if and outside the if - dont use let, only if you really have to

Reply:
fixed